infrastructure/base/modules/env/main.tf

```diff
@@ -0,0 +1,72 @@
+module "ecr" {
+  source = "../ecr"
+
+  project     = var.project
+  environment = var.environment
+  tags = {
+    project     = var.project,
+    environment = var.environment
+  }
+}
+
+
+resource "aws_security_group" "postgresql_access" {
+  vpc_id      = var.vpc.id
+  description = "SG allowing access to the Postgres SG"
+
+  tags = merge(
+    {
+      Name = "EC2 SG to access RDS - ${var.environment}"
+    },
+    var.tags
+  )
+}
+
+resource "aws_security_group_rule" "port_forward_postgres" {
+  type                     = "egress"
+  from_port                = module.postgresql.port
+  to_port                  = module.postgresql.port
+  protocol                 = "-1"
+  source_security_group_id = module.postgresql.security_group_id
+  security_group_id        = aws_security_group.postgresql_access.id
+}
+
+module "postgresql" {
+  source = "../postgresql"
+
+  log_retention_period        = var.rds_log_retention_period
+  subnet_ids                  = var.subnet_ids
+  project                     = var.project
+  environment                 = var.environment
+  rds_backup_retention_period = var.rds_backup_retention_period
+  rds_user_name               = "postgres"
+  rds_engine_version          = var.rds_engine_version
+  rds_instance_class          = var.rds_instance_class
+  rds_instance_count          = var.rds_instance_count
+  tags                        = var.tags
+  vpc_id                      = var.vpc.id
+  rds_port                    = 5432
+  vpc_cidr_block              = var.vpc.cidr_block
+  availability_zones          = var.availability_zones
+  database_name               = var.project
+}
+
+module "beanstalk" {
+  source = "../beanstalk"
+
+  project                 = var.project
+  environment             = var.environment
+  region                  = var.aws_region
+  application_name        = "${var.project}-${var.environment}"
+  application_environment = "${var.project}-${var.environment}-environment"
+  solution_stack_name     = "64bit Amazon Linux 2 v3.6.0 running Docker"
```

Comment:
not sure if this is a description for human eyes, or a config value that actually does something behind the scenes. If it's the latter, it might be a good thing to set as a var

Reply:
It is a config option, it's just that I thought it is not very useful to set it as var, because you cannot change it to anything. It is the only stack on which this will run from those available today, and if tomorrow an upgraded version will appear it is quite likely that more changes will be needed to the deployment process than just changing this string. So I kind of didn't want to encourage changing this string :) But can be done with proper description.